examples-gallery/plot_ball_rolling_on_spinning_disc.py

```diff
@@ -0,0 +1,620 @@
+ # %%
+"""
+Ball Rolling on Spinning Disc
+=============================
+
+A uniform, solid ball with radius r and mass :math:`m_b` is rolling on a
+horizontal spinning disc without slipping.
+The disc starts at rest and speeds up like this
+:math:`u_{3}(t) = \\Omega \\cdot (1 - e^{(-\\alpha \\cdot t)})`
+where :math:`\\alpha` > 0 is a measure of the acceleration and :math:`\\Omega`
+is the final rotational speed.
+A torque is applied to the ball, and the goal is to get it to the
+center of the disc.
+
+An observer, a particle of mass :math:`m_o`, is attached
+to the surface the ball.
+
+**Constants**
+
+- :math:`m_b`: mass of the ball [kg]
+- r : radius of the ball [m]
+- :math:`m_o` : mass of the observer [kg]
+- :math:`\\Omega`: final rotational speed of the disc
+  around the vertical axis [rad/sec]
+- :math:`\\alpha`: measure of the acceleration of the disc [1/sec]
+
+**States**
+
+- :math:`q_1, q_2, q_3`: generalized coordinates of the ball
+    w.r.t the disc [rad]
```

Comment:
```suggestion
   w.r.t the disc [rad]
```

3 spaces for indentation in RestructuredText